blockchain/state/state_object.go

```diff
@@ -257,15 +257,27 @@ func (self *stateObject) updateStorageTrie(db Database) Trie {
 	return tr
 }
 
-// UpdateRoot calls updateStorageTrie to get the latest root hash of the object's storage trie
+// updateStorageRoot calls updateStorageTrie to get the latest root hash of the object's storage trie
```

Comment:
Please modify this comment. `updateStorageRoot` doesn't call `updateStrageTrie` anymore.